src/Search/Search.js

```diff
@@ -193,23 +213,28 @@ class Search extends Component {
               </div>
             </div>
             <div className="pure-u-md-1-6">{periodSelect}</div>
-            {this.showMissingPrisonersButton() && (
-              <Flag
-                name={['updateAttendanceEnabled']}
-                render={() => (
-                  <MissingButtonContainer>
-                    <Button
-                      buttonColour={BLUE}
-                      onClick={() => history.push(`missing-prisoners`)}
-                      data-qa="missing-prisoners"
-                    >
-                      View missing prisoners
-                    </Button>
-                  </MissingButtonContainer>
-                )}
-                fallbackRender={() => <></>}
-              />
-            )}
+            <ButtonsContainer>
+              {this.showMissingPrisonersButton() && (
+                <Flag
+                  name={['updateAttendanceEnabled']}
+                  render={() => (
+                    <MissingButtonContainer>
+                      <Button
+                        buttonColour={BLUE}
+                        onClick={() => history.push(`missing-prisoners`)}
+                        data-qa="missing-prisoners"
+                      >
+                        View missing prisoners
+                      </Button>
+                    </MissingButtonContainer>
+                  )}
+                  fallbackRender={() => <></>}
+                />
+              )}
+              <StatsLink href="/attendance-reason-statistics" id="attendanceStatsButton">
```

Comment:
does this link need an `id` - think should be `data-qa` if used for testing.  Also doesn't need to be called a button when it isn't!